packages/components/tree-select/src/cache-options.ts

```diff
@@ -0,0 +1,26 @@
+import { defineComponent, inject } from 'vue'
+import { selectKey } from '@element-plus/components/select'
+import type { SelectContext } from '@element-plus/components/select'
+import type { PropType } from 'vue'
+
+export type CacheOption = {
```

Comment:
Maybe the following type is better 🤔
```
export type CacheOption = {
  value: string | number | Record<string, string>
  label: string | number
  isDisabled: boolean
}
```

Reply:
Yes, any type is bad, has been modified to be consistent with el-option instances.

https://github.com/element-plus/element-plus/pull/10716/commits/3fd07a2c9c7e6c4338486b52bb95e9cbcadaf1e2

Reply:
https://github.com/element-plus/element-plus/pull/10716/commits/52ae3a3fe342c82f9cf7fcf21bd091a4d4f32d61

Reply:
Yes, any type is bad, has been modified to be consistent with el-option instances.

https://github.com/element-plus/element-plus/pull/10716/commits/3fd07a2c9c7e6c4338486b52bb95e9cbcadaf1e2